iceoryx_examples/icecubetray/topic_data.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+#ifndef IOX_EXAMPLES_ICECUBETRAY_TOPIC_DATA_HPP
+#define IOX_EXAMPLES_ICECUBETRAY_TOPIC_DATA_HPP
+
+#include <cstdint>
+
+struct RadarObject
+{
+    RadarObject() noexcept
```

Comment:
Why do we need the default ctor? I think it should be removed.

Reply:
`iox::popo::Publisher::loan()` requires a default c'tor as it is default constructing it. The `RadarObject` should be consistent throughout all examples.